ocaml/graph/go

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
+set -e
+
+GRAPH="../../graph.native -h $1 -u $2 -pw $3"
 DOTARGS="-Tpng -Ktwopi -Goverlap=false"
-GRAPH=./graph
 
 ${GRAPH} VM VBD VDI SR > vm-disks.dot
 ${GRAPH} VM VIF network > vm-network.dot
 ${GRAPH} host PIF network > host-network.dot
 
-for i in vm-disks.dot vm-network.dot host-network.dot
+for i in vm-disks vm-network host-network
 do
-  dot ${DOTARGS} $i > $i.png
+  dot ${DOTARGS} $i.dot > $i.png
 done
+
+set +e
```

Comment:
This is simply to restore the shell state in the case the script is sourced (if anyone should want to source it that is!). I think it's a good practice to unset a flag if I have explicitly set it.